spec/index.bs

```diff
@@ -114,33 +168,131 @@ interface GPUAdapter : GPUObjectBase {
 };
 </script>
 
+{{GPUAdapter}} has the following attributes:
+
+<dl dfn-type=attribute dfn-for=GPUAdapter>
+    : <dfn>\[[adapter]]</dfn>, of type [=adapter=], readonly
+    ::
+        An internal slot holding the [=adapter=] to which this {{GPUAdapter}}
+        refers.
+
+    : <dfn>name</dfn>
+    ::
+        A human-readable name identifying the adapter.
+        The contents are implementation-defined.
+
+    : <dfn>extensions</dfn>
+    ::
+        A {{GPUExtensions}} object which enumerates the extensions supported
+        by the user agent, and whether each extension is supported by the
+        underlying implementation.
+          - If an extension is not supported by the user agent,
+            it will not be present in the object.
+          - If an extension is supported by the user agent, but
+            not by the [=adapter=], it will be `false`.
+          - If an extension is supported by the user agent and
+            by the [=adapter=], it will be `true`.
+</dl>
+
 ### Getting an Adapter ### {#adapter-creation}
 
-<script type=idl>
-[Exposed=Window]
-interface GPU {
-    // May reject with DOMException  // TODO: DOMException("OperationError")?
-    Promise<GPUAdapter> requestAdapter(optional GPURequestAdapterOptions options);
-};
-</script>
+To get a {{GPUAdapter}}, use {{GPU/requestAdapter()}}.
 
 <script type=idl>
 dictionary GPURequestAdapterOptions {
-    GPUPowerPreference? powerPreference;
+    GPUPowerPreference powerPreference;
```

Comment:
if it's not `?` optional, shouldn't we assign a default value here?

Reply:
Like in WebGL I think there would be three possible values here: `"low-power"`, `"high-performance"`, and 'default'. Right now 'default' is represented by `undefined`. Should we add `"default"` as a member of the enum and make it the actual explicit default? Should we get rid of that and just make the default `"low-power"`?

Reply:
I think the "default" is still useful, i.e. you are on a multi-GPU system, and UA can pick the discrete GPU if it sees that the power is plugged in.
Saying that, thanks for clarifying the removal of "?". The change is good.